ext/hash/blake3/fetch_upstream_blake3.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
this file should be added to https://github.com/php/php-src/blob/php-8.3.2/.github/actions/verify-generated-files/action.yml to assert the upstream files are not unintentionally modified

Reply:
@mvorisek like, just add it to the end of the script?
```
        build/gen_stub.php -f
        build/gen_stub.php --generate-optimizer-info
        git add . -N && git diff --exit-code
+      ext/hash/blake3/fetch_upstream_blake3.sh
```
like that?

Reply:
added: fcc16d318a1eebab7938c1421b7df3e92f0e5198

Reply:
That does not seem necessary. We don't do that for any other dependencies.